contributed_definitions/nyaml/NXresolution.yaml

```diff
@@ -0,0 +1,42 @@
+category: base
+doc: |
+  Describes the resolution of a physical quantity.
+type: group
+NXresolution:
+  dimension:
```

Comment:
This is not really what is meant by dimension, right? Maybe "quantity" or "measure" are better?